src/database/type-transforms.tsx

```diff
@@ -5,7 +5,7 @@ import { PlannedRelease } from "../impact-dashboard/EpidemicModelContext";
 import { Facility, ModelInputs, Scenario } from "../page-multi-facility/types";
 
 const timestampToDate = (timestamp: firebase.firestore.Timestamp): Date => {
-  return timestamp.toDate();
+  return timestamp?.toDate() || new Date();
```

Comment:
@jessex - this fixes that seeming race condition you and I were discussing earlier.  I don't think this is an acceptable solution and therefore this should be considered a merge blocker.  I'm working on figuring out a better solution now

Reply:
I agree. Thanks for looking into this!